### ISSUE TITLE
embedded images don't print?

### ISSUE THREAD

**Issue by jasper-cs** [MrTarantula/WIPrint#18]
some work items have images included in the Description tab, but these don't appear when I try to print them.  is there a way to enable this?

**Comment by MrTarantula** [comment]
v2.2.3 adds a small delay so the browser can try to fetch images before printing. Please test and let me know if it works for you.

**Comment by jasper-cs** [comment]
hello -- sadly, doesn't quite work.  see attached pdf -- you'll see a 
bit of an image on page 2

**Comment by MrTarantula** [comment]
I could make the delay longer. I don't like that solution though. I'd rather find a way to wait until any and all content is loaded, and no longer. I should have a fix soon.

**Comment by jasper-cs** [comment]
thanks for your efforts - will be super helpful

**Comment by MrTarantula** [comment]
Is this still an issue? I've made a few updates since then.

**Comment by cloudy81** [comment]
I have tested with version 2.3.0. Images in several fields as well in comments will be printed as PDF. 

If you select "Attached file" on the work item type nothing will be printed/exported.

**Comment by MrTarantula** [comment]
Yes, attached files are a whole other issue. Not sure I can get attached images in a timely manner to print. We'll see.

**Comment by cloudy81** [comment]
So for me that issue looks as resolved. Because embedded images are printed
